src/components/Disclaimer/Consent.tsx

```diff
@@ -0,0 +1,123 @@
+                  <Dialog.Title
+                    as="h3"
+                    className="text-2xl font-medium leading-6 text-white"
+                  >
+                    DISCLAIMER
+                  </Dialog.Title>
+                  <div className="mt-6">
+                    <p className="mb-5 text-sm leading-6 text-slate-300">
+                      Fortress Finance is a set of smart contracts made
+                      available by Fortress.Finance on a voluntary, “as-is” and
+                      “as available” basis. By interacting or attempting to
+                      interact (in both cases, “interacting”) with Umami, you
+                      confirm that you understand and agree to these terms:
+                    </p>
+                    <p className="mb-5 text-sm leading-6 text-slate-300">
+                      Nature of Finance Fortress: The protocol is completely
+                      decentralized, Fortress.Finance is not involved in any
+                      transactions, whether as an intermediary, counterparty,
+                      advisor or otherwise.
+                    </p>
+                    <p className="mb-5 text-sm leading-6 text-slate-300">
+                      You are not a US Person; you are not a resident, national,
+                      or agent of Antigua and Barbuda, Algeria, Bangladesh,
+                      Bolivia, Belarus, Burundi, Burma (Myanmar), Cote D’Ivoire
+                      (Ivory Coast), Crimea and Sevastopol, Cuba, Democratic
+                      Republic of Congo, Ecuador, Iran, Iraq, Liberia, Libya,
+                      Magnitsky, Mali, Morocco, Nepal, North Korea, Somalia,
+                      Sudan, Syria, Venezuela, Yemen, Zimbabwe or any other
+                      country to which the United States, the United Kingdom or
+                      the European Union embargoes goods or imposes similar
+                      sanctions; you are not a member of any sanctions list or
+                      equivalent maintained by the United States government, the
+                      United Kingdom government, the European Union, or the
+                      United Nations; you do intend to transact with any
+                      Restricted Person or Sanctions List Person; you do not,
+                      and will not, use VPN software or any other privacy or
+                      anonymization tools or techniques to circumvent, or
+                      attempt to circumvent, any restrictions.
+                    </p>
+                  </div>
+
+                  <div className="mt-4">
+                    <Button
+                      className="focus-visible:ring-blue-500 inline-flex w-full justify-center px-4 py-2 text-sm font-medium focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2"
```

Comment:
I don't think your overrides on focus styles are actually doing anything here. 

I believe you did this because, when the modal opens, the button is automatically focused. This is a feature with HeadlessUI. Those focus states will be shown with `focus-visible`.